doc/nrf/protocols/matter/end_product/platform_dmp_cert.rst

```diff
@@ -0,0 +1,395 @@
+
+Recertification scenarios
+=========================
+
+Use the following scenarios to determine when recertification is required:
+
+* No changes (platform, product, spec) - No recertification.
+* Product-only changes (same platform software, same spec) - DMP recertification with testing (Product tests).
+* Platform-only software change (same spec) with platform recertified - DMP recertification, no DMP testing required (documentation update).
+* Platform software change (same spec) and product changes - DMP recertification with testing (Product tests).
+* Spec version change without updated platform software - Not allowed: the platform must update first.
+* Spec version change with updated or recertified platform software:
+
+  * If product is unchanged - Spot-check of Product tests: platform must have been recertified.
+  * If product also changes - DMP recertification with testing (Product tests).
+
+Use the following guidance when planning changes:
+
+* Widening capabilities beyond the platform SOE requires a platform update and recertification before DMP can proceed.
+* Changing optional features within platform scope leads to a focused DMP test run for the affected areas and the Product tests.
+
+.. _ug_matter_dmp_ncs_guideline:
+
+Guidelines for leveraging the platform's |NCS| to build a DMP
+*************************************************************
+
+This section provides practical steps for using the |NCS| to move from a platform to a DMP.
+
+.. rst-class:: numbered-step
+
+Choose a supported configuration
+================================
+
+Start by reviewing the platform's SOE and selecting a supported SoC and transport.
+Use the specific Matter component within the |NCS|, which corresponds to the platform certification.
+
+.. rst-class:: numbered-step
+
+Fetch sources and build a reference certification template
+==========================================================
+
+Initialize at the platform's certified |NCS| tag and build the certification template as a reference for what the platform enabled during platform testing:
+
+.. code-block:: console
+
+   west init -m https://github.com/nrfconnect/sdk-nrf --mr <platform_certified_tag>
+   west update
+
+   # The reference data model used during platform testing can be found here:
+   # samples/matter/common/src/certification
+   # Build the certification template (reference) to inspect platform-enabled features
+   west build -b <board> samples/matter/template -T sample.matter.template.certification
+   west flash
+
+This template is provided as a reference (transport, core clusters).
+You may customize your product by enabling or disabling optional clusters, attributes, and features and by building your own application data model.
+When customizing, ensure that your PICS reflects the final feature set, remain within the platform’s SOE (boards, radios and SDK/OS versions), and do not alter the core platform functionality covered by the platform certification.
```

Comment:
Can we link once again here: Matter nRF Connect SDK Platform Certificate and mention that our PICS are linked there alongside of the certificate?